src/components/RichTextEditor/plugins/linkChooserPlugin/transforms/getAndUpsertLink.ts

```diff
@@ -0,0 +1,61 @@
+import { getAbove, getPluginType, isCollapsed, PlateEditor, unwrapNodes } from "@udecode/plate-core";
+import { IconEnum } from "../../../../..";
```

Comment:
```suggestion
import { IconEnum } from "@foundation/Icon";
```
So it's more clear where this is coming from.